conf/machine/freedom-u540.conf

```diff
@@ -18,15 +18,20 @@ QEMUVERSION = "riscv"
 
 EXTRA_IMAGEDEPENDS += "riscv-pk"
 
-SERIAL_CONSOLE = "115200 ttyS0"
+SERIAL_CONSOLE = "115200 console"
 
 MACHINE_EXTRA_RRECOMMENDS += " kernel-modules"
 
-RISCV_BBL_PAYLOAD ?= "${KERNEL_IMAGETYPE}${KERNEL_INITRAMFS}-${MACHINE}.bin"
```

Comment:
you can keep this as it is. Since KERNEL_INITRAMFS is not defined it will be reduced to what you expect it to be ${KERNEL_IMAGETYPE}-${MACHINE}.bin